### ISSUE TITLE
Non-hardwired text style

### ISSUE THREAD

**Issue by deakjahn** [fayaz07/progress_dialog#12]
This is a question of tastes, for sure, but I wouldn't hardwire the font color and size but would use what the platform provides instead, eg:

    child: Text(_dialogMessage, style: Theme.of(context).textTheme.subhead),
    child: Text("$_progress/100", style: Theme.of(context).textTheme.caption),
    child: Text(message, textAlign: TextAlign.center, style: Theme.of(context).textTheme.subhead),

**Comment by fayaz07** [comment]
So, you suggest me to use these styles?

**Comment by deakjahn** [comment]
Yes. I do. Pick one that you consider nice but don't specify size and color manually. That's my suggestion.

**Comment by christianwico** [comment]
Why not just let it be fully-extensible? The default theme is okay but some users would want to customize the dialog entirely...

Something like:
```dart
...

progressDialog.show(
   message,
   indicator: customIndicatorWidget,
   textStyle: Theme.of(context).primaryTextTheme.title,
   dialogTheme: DialogTheme(
    shape: RoundedRectangleBorder(
      side: BorderSide(
        color: Color(0xff000000),
        width: 0.0,
        style: BorderStyle.none,
      ),
    ),
  ),
);

...
```

**Comment by fayaz07** [comment]
That's a cool idea! @HakuFlou

**Comment by fayaz07** [comment]
Fixed in latest update pub v1.2.0, update will be published soon! Thank you!
